Src/VimCore/CommandUtil.fs

```diff
@@ -1084,6 +1086,22 @@ type internal CommandUtil
         let span = action snapshotData
         BufferGraphUtil.MapSpanDownToSingle _bufferGraph span x.CurrentSnapshot
 
+    /// Extend the selection for a mouse click
+    member x.ExtendSelectionForMouseDrag (visualSpan: VisualSpan) =

```

Comment:
>(visualSpan: VisualSpan) [](start = 41, length = 24)

Why does this take a VisualSpan and not use it?

Reply:
At one point it manipulated the visual span, but eventually I realized it is like any other caret motion that simply affects what is selected. Will remove.